### DISCUSSION TITLE
importing react three drei shows error on home page: Expression produces a union type that is too complex to represent. (ts & mui)

### DISCUSSION THREAD

**Discussion by Achi00** [pmndrs/drei#1419] [Q&A]
hi i am creating react js website using typescript and mui and I have 3d component on another file, as soon as I import react three drei it shows error on home pages mui sx ={{}} I have installed types/three but still cant fix, what may cause the error?
thanks

**Comment by MoonStation8** [comment]
I get that error if I don't specify the component type. For example `<Box component="div">` all good. Just `<Box>` throws an error.
